README.md

```diff
@@ -88,9 +88,9 @@ The projects in the [`deployment-platforms`](./deployment-platforms) directory s
 
 ## About this repository
 
-The `master` branch of this repository contains the examples with the latest stable version of Prisma CLI and Prisma Client. These dependencies are kept up to date with a GitHub Action workflow, that updates them every time a new version of Prisma is released.
+The `master` branch of this repository contains the examples with the latest stable version of Prisma CLI and Prisma Client. These dependencies are kept up to date with a GitHub Action workflow, which updates them every time a new version of Prisma is released.
 
-There also is a `alpha` branch that mirrors the code from `master` (synced via a GitHub Action workflow), but uses the latest unstable version of Prisma CLI and Prisma Client instead (also updated via a GitHub Action workflow). Thanks to the test coverage of all projects, this can point us to incompatibilities early.
+There are also the automated branches `dev` and `patch-dev`, which mirror the code from `master` (synced via a GitHub Action workflow), but they use the respective development channels of Prisma CLI and Prisma Client from npm instead (also updated via a GitHub Action workflow). Thanks to the test coverage of all projects, this can point us to incompatibilities early.
```

Comment:
```suggestion
There are also the automated branches `dev` and `patch-dev`, which mirror the code from `master` (synced via a GitHub Action workflow), but they use the respective development channels of Prisma CLI and Prisma Client from npm instead (`@dev` and `@patch-dev`, also updated via a GitHub Action workflow). Thanks to the test coverage of all projects, this can point us to incompatibilities early.
```